src/SourceBuild/content/eng/pipelines/vmr-license-scan.yml

```diff
@@ -111,23 +115,14 @@ extends:
             artifactName: $(Agent.JobName)_BuildLogs_Attempt$(System.JobAttempt)
         steps:
 
-        - script: |
-            source ./eng/common/tools.sh
-            InitializeDotNetCli true
-          displayName: Install .NET SDK
-          workingDirectory: $(Build.SourcesDirectory)
-
         - task: PipAuthenticate@1
           displayName: 'Pip Authenticate'
           inputs:
             artifactFeeds: public/dotnet-public-pypi
             onlyAddExtraIndex: false
 
-        - script: $(sdkRoot)/eng/install-scancode.sh
```

Comment:
Is anything still using this file, can it be removed?

Reply:
I searched for the keyword "scancode" in the source build repository and found that only the license scan test is using this toolkit.

Reply:
Does that confirm that it can be removed with this PR?

Reply:
I thought I had removed this file in the previous update to the PR, but it looks like I missed it. I 've now included the update: https://github.com/dotnet/sdk/pull/48622/commits/f18dd2ff28b428a7bb371f01c63738a11fae715c